### DISCUSSION TITLE
🖥️  Apple Silicon Powered M1 macOS Larger Runners 🚢

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#69096] [Actions]
![Screenshot 2023-10-03 at 3 42 56 PM](https://github.com/community/community/assets/95834130/05739e17-3e1c-4689-8de9-370f484fac7d)

**[Detailed Changelog](https://github.blog/2023-10-02-introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/)**

### 🖥️ Apple Silicon Powered M1 macOS Larger Runners

Apple developers need the latest chipset to utilize features in iOS and macOS. The M1 macOS runner, with GPU hardware acceleration enabled by default, offers improved performance and efficiency. It significantly reduces build times compared to Intel runners.

### 📱 How GitHub Uses the M1 Runner to Build GitHub Mobile for iOS

The GitHub iOS team leverages the M1 runner for over 10k minutes weekly, accelerating updates for the GitHub iOS app on the Apple App Store.

### ⚙️ How to Use the Runner

Update the `runs-on` key in your GitHub Actions YAML workflow to target `macos-latest-xlarge` or `macos-13-xlarge`. The transition is seamless, offering substantial build time improvements.

**There is no sign-up required for the beta and the runner is immediately available to all developers, teams, and enterprises.**

![Screenshot of updating the `runs-on` key](https://github.blog/wp-content/uploads/2023/09/actions-quickstart.png?w=549&resize=549%2C304)

### 💲 New macOS Runner Pricing

We have decreased the price of macOS larger runners, now priced at $0.16/minute for XL and $0.12/minute for large.

[Learn more about pricing](https://docs.github.com/billing/managing-billing-for-github-actions/about-billing-for-github-actions#per-minute-rates).

### 🛣️ What’s Next?

Track progress towards the general availability of macOS larger runners [here](https://github.com/github/roadmap/issues/819). Stay updated on the [GitHub blog](https://github.blog/) for the latest GitHub Actions updates.

**Comment by anil0906** [comment]
Is there a m1 support for smaller runners? Like macos-13-arm64 ?

Reference:
https://github.com/actions/runner-images/blob/main/images/macos/macos-13-arm64-Readme.md

**Reply by kgantchev** [reply]
@anil0906 [FlyCI](https://www.flyci.net/?utm_source=github&utm_medium=community&utm_campaign=recommend) offers smaller runners (as well as M2 runners). The FlyCI runners are more affordable than GitHub's. Feel free to give them a shot!

The usage is simple: 

Step 1: Install the [FlyCI GitHub app](https://www.flyci.net/get-started/?utm_source=github&utm_medium=community&utm_campaign=recommend). 
Step 2: Give the FlyCI app permissions to the repos for which you would like to use the FlyCI MacOS runners. 
Step 3: Change the `runs-on` flag to use the FlyCI label (see below). 

```diff
jobs:
 ci:
-    runs-on: macos-latest
+    runs-on: flyci-macos-large-latest-m2
   steps:
   - name: 👀 Checkout repo
     uses: actions/checkout@v4
```
